src/components/internal/SecuredFields/utils.js

```diff
@@ -75,25 +75,30 @@ export const getErrorReducer = (numDateFields, state) => (acc, field) => {
 /**
  * Create an object suitable for sending to our handleOnError function
  */
-export const getErrorObject = (fieldType, rootNode, state) => ({
-    rootNode,
-    fieldType,
-    error: getProp(state, `errors.${fieldType}`) || DEFAULT_ERROR,
-    type: 'card'
-});
+export const getErrorObject = (fieldType, rootNode, state) => {
+    const error = getProp(state, `errors.${fieldType}`) || DEFAULT_ERROR;
+    //    console.log('\n### utils::getErrorObject:: error =', error);
+
+    return {
+        rootNode,
+        fieldType,
+        error,
+        type: 'card'
+    };
+};
 // -- end ROUTINES USED IN SecuredFieldsProvider.showValidation -----------------------
 
 /**
  * Used below in addErrorTranslationToObject (called from SecuredField.ts) AND also by handler for SecuredFieldComponent aka CustomCardComponent
  */
-export const getTranslatedErrors = (i18n = {}) => ({
-    [ENCRYPTED_CARD_NUMBER]: i18n.get && i18n.get('creditCard.numberField.invalid'),
-    [ENCRYPTED_EXPIRY_DATE]: i18n.get && i18n.get('creditCard.expiryDateField.invalid'),
-    [ENCRYPTED_EXPIRY_MONTH]: i18n.get && i18n.get('creditCard.expiryDateField.invalid'),
-    [ENCRYPTED_EXPIRY_YEAR]: i18n.get && i18n.get('creditCard.expiryDateField.invalid'),
-    [ENCRYPTED_SECURITY_CODE]: i18n.get && i18n.get('creditCard.oneClickVerification.invalidInput.title'),
-    defaultError: 'error.title'
-});
+//export const getTranslatedErrors = (i18n = {}) => ({
```

Comment:
It looks like this is not necessary anymore, shall we remove it?

Reply:
I thought it was gone already... looks like I didn't commit it

Reply:
done